pages/account_and_service_management/account_information/security_certifications/guide.en-gb.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "OVHcloud products ISO27k certifications"
+updated: 2023-12-20
+---
+
+## Objectifs
+
+The following page aims at presenting OVHcloud products covered by ISO/CEI 27k family norms.
+
+## Norm's versions
+
+Our actual certificates are established following these norms and versions:
+
+- ISO/IEC 27001:2022
+- ISO/IEC 27017:2015
+- ISO/IEC 27018:2019
+- ISO/IEC 27701:2019
+
+We watch the new versions of ISO27017, ISO27018 and ISO27701 actually under revision by ISO.
+
+## Actual certified OVHcloud products
+
+| **Product** | **Certified** | 
+| --- | ---  | 
+| Dedicated Server | yes | 
+| Nutanix on OVHcloud | yes | 
+| Enterprise File Storage | yes | 
+| Block Storage | yes | 
+| VMware on OVHcloud | yes | 
+| Private Cloud SecNumCloud | yes | 
+| NAS-HA | yes | 
+| OpenStack Instances | yes | 
+| AI Deploy | yes | 
+| AI Notebooks | yes | 
+| AI Training | yes | 
+| Data Processing | yes | 
+| Managed Kubernetes | yes |
+| Search engine software platform | ETA : 01/2024 | 
+| Timeseries | ETA : 01/2024 |
+| NoSQL Database | ETA : 01/2024 | 
```

Comment:
hello, 
last modification
for column certified ==> to be changed  nad put : 
MongoDB : yes
Others : ETA  01/2024